packages/js/test/plugins/auctionHouseModule/updateAuctionHouse.test.ts

```diff
@@ -115,3 +116,78 @@ test('[auctionHouseModule] it throws an error if nothing has changed when updati
   // Then we expect an error.
   await assertThrows(t, promise, /No Instructions To Send/);
 });
+
+test('[auctionHouseModule] delegate Auctioneer on Auction House update.', async (t) => {
+  // Given an existing Auction House.
+  const mx = await metaplex();
+  const auctioneerAuthority = Keypair.generate();
+
+  const { auctionHouse } = await mx
+    .auctions()
+    .createAuctionHouse({ sellerFeeBasisPoints: 200 })
+    .run();
+
+  t.false(auctionHouse.hasAuctioneer);
+
+  // When we send an update without providing any changes but with auctioneerAuthority to delegate.
+  const { auctionHouse: updatedAuctionHouse } = await mx
+    .auctions()
+    .updateAuctionHouse(auctionHouse, { auctioneerAuthority })
+    .run();
+
+  t.ok(updatedAuctionHouse.hasAuctioneer);
+
+  // Auctioneer was delegated.
+  const ahAuctioneerPda = findAuctioneerPda(
+    auctionHouse.address,
+    auctioneerAuthority.publicKey
+  );
+  const ahAuctioneerAccount = await mx.rpc().getAccount(ahAuctioneerPda);
+  t.ok(ahAuctioneerAccount.exists);
+});
+
+test('[auctionHouseModule] it throws an error if delegates different Auctioneer Authority on Auctioneer Auction House update.', async (t) => {
+  // Given an existing Auction House.
+  const mx = await metaplex();
+  const auctioneerAuthority = Keypair.generate();
+  const secondAuctioneerAuthority = Keypair.generate();
+
+  // Create Auction House and delegate auctioneerAuthority.
+  const { auctionHouse } = await mx
+    .auctions()
+    .createAuctionHouse({ auctioneerAuthority, sellerFeeBasisPoints: 200 })
+    .run();
+
+  t.ok(auctionHouse.hasAuctioneer);
+
+  // When we send an update without providing any changes but with different auctioneerAuthority to delegate.
+  const promise = mx
+    .auctions()
+    .updateAuctionHouse(auctionHouse, {
+      auctioneerAuthority: secondAuctioneerAuthority,
+    })
+    .run();
```

Comment:
As mentioned on a different comment, this should be a valid scenario since it is possible to update the auctioneer instance of an Auction House.